pkg/util/pretty/prefix_text_printer.go

```diff
@@ -5,8 +5,9 @@ import "github.com/pterm/pterm"
 // Pretty prefix text printer style.
 //
 // Usage:
-//   pretty.InfoT.Println("Hello")
-//   pretty.WarningT.Println("Hello")
+//
```

Comment:
double check: https://pkg.go.dev/kusionstack.io/kusion@v0.7.2/pkg/util/pretty

![image](https://user-images.githubusercontent.com/23924124/206347477-3fd75b1e-4d27-44e9-a108-745d0cf0f2b5.png)

Reply:
Same format issue. It is recognized as blocker format issue in golangci-lint v1.50.1.